GTG/gtk/browser/main_window.py

```diff
@@ -990,6 +990,27 @@ def on_add_subtask(self, widget=None):
 
             self.app.open_task(task.get_id(), new=True)
 
+    def on_add_parent(self, widget=None):
+        uid = self.get_selected_task()
+        selected_task = self.req.get_task(uid)
+        if uid:
+            parents = selected_task.get_parents()
+            if parents:
+                # Switch parents
+                for p_tid in parents:
+                    par = self.req.get_task(p_tid)
+                    if (par.is_loaded() and par.get_status() in
+                        (Task.STA_ACTIVE)):
```

Comment:
Why not `par.get_status() == Task.STA_ACTIVE`?
Also, from what I understand, it just checks whenever the status is contained in `Task.STA_ACTIVE`, not equal to it (`(Task.STA_ACTIVE) == Task.STA_ACTIVE`, you probably thought of `(Task.STA_ACTIVE,)`, which is a tuple)

Reply:
Yeah, that should be `par.get_status() == Task.STA_ACTIVE`.

>  you probably thought of (Task.STA_ACTIVE,)

yep
However, I don't know why there are these ifs in the code base (I was just mimicking them)
https://github.com/getting-things-gnome/gtg/blob/d7bbcd5b2a9d7d54b2b28278e5bbf6fd264b302b/GTG/core/task.py#L213
wouldn't be best to have `c.get_status() == self.STA_ACTIVE`